assets/src/application/ui/styles/margins.scss

```diff
@@ -0,0 +1,51 @@
+.ee-horizontal-margin {
+    &--nano {
+        margin-left: var(--ee-margin-nano);
+        margin-right: var(--ee-margin-nano);
+    }
```

Comment:
this can be improved with SASS foreach, but didn't wanted to spend too much time on this now